.github/workflows/examples-infineon.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Infineon P6
+
+on:
+    push:
+    pull_request:
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true    
+
+jobs:
+    infineon:
+        name: Infineon examples building
+        timeout-minutes: 30
+        
+        env:
+            BUILD_TYPE: P6
+            
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        container:
+            image: connectedhomeip/chip-build-infineon:latest
+            volumes:
+                - "/tmp/output_binaries:/tmp/output_binaries"
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+
+            - name: Build lock-app example
+              timeout-minutes: 10
+              run: scripts/examples/gn_p6_example.sh
+                   examples/lock-app/p6 out/lock_app_p6
+
+            - name: Copy aside build products
```

Comment:
I believe we have sufficient size checks, lets skip copying and adding build artifacts here. Our storage size is limited and removing the binary artifact will make the workflow a lot simpler.

Reply:
@andy31415 : thanks for your review comments.
Added bootstrap step and removed copy/adding build artifacts steps